src/libecalc/dto/components.py

```diff
@@ -190,8 +190,17 @@ class PumpSystemOperationalSetting(EcalcBaseModel):
     outlet_pressures: List[Expression]
 
 
+class SystemCrossoverStream(EcalcBaseModel):
```

Comment:
Will this not be the same all places `Crossover` is used, so just call it `Crossover`?